msbuild/Xamarin.iOS.Tasks.Core/Tasks/CompileAppManifestTaskBase.cs

```diff
@@ -108,7 +108,15 @@ bool Compile (PDictionary plist)
 			var dtSettings = currentSDK.GetDTSettings ();
 
 			SetValue (plist, ManifestKeys.BuildMachineOSBuild, dtSettings.BuildMachineOSBuild);
-			plist.SetIfNotPresent (ManifestKeys.CFBundleDevelopmentRegion, "en");
+			// We have an issue here, this is for consideration by the platform:
+			// CFLocaleCopyCurrent(), used in the mono code to get the current locale (locale.c line 421), will return the value of the application's CFBundleDevelopmentRegion Info.plist key if all of the following conditions are true:
```

Comment:
Minor: please use a permanent link to the code in locale.c instead of pointing to the line number, because the line number will change in the future (see https://github.com/blog/2415-introducing-embedded-code-snippets)

Reply:
Fair point, will do once we agree of the correct fix. locale.c Does use CFLocaleCopyCurrent, that is our main issue atm. Could be a regression as I mentioned in my other comment.